### PR TITLE
Copenhagen is parented by Denmark not Sweden! 🚨

### PR DESCRIPTION
**Pull request by orangejulius**
We would like to prevent any international incidents: https://twitter.com/peterbrodersen/status/723404317152010241

Also note this is technically Hovedstaden being updated, which is the Copenhagen region or something.

### CODE REVIEW

**Comment by orangejulius** [comment]
Oops, this won't fix things by itself. The heirarchy for all the children of this record should be updated too, right?

**Comment by thisisaaronland** [comment]
The tl;dr is: "Skåne", specifically "Skåne" before we gave precedence to `lbl:` centroids (derived by Mapshaper) over `geom:` centroids.

The long version follows:

Copenhagen, reverse geocoded using `lbl:` coordinates:

```
curl -s 'http://localhost:1111/country?latitude=55.704438&longitude=12.502119' | python -mjson.tool
[
    {
        "Id": 85633121,
        "Name": "Denmark",
        "Offset": -1,
        "Placetype": "country"
    }
]
```

Copenhagen, reverse geocoded using `geom:` coordinates:

```
curl -s 'http://localhost:1111/country?latitude=55.921414&longitude=12.318814' | python -mjson.tool
[
    {
        "Id": 85633121,
        "Name": "Denmark",
        "Offset": -1,
        "Placetype": "country"
    }
]
```

Hovedstaden, reverse geocoded using `lbl:` coordinates:

```
curl -s 'http://localhost:1111/country?latitude=55.697507&longitude=12.490749' | python -mjson.tool
[
    {
        "Id": 85633121,
        "Name": "Denmark",
        "Offset": -1,
        "Placetype": "country"
    }
]
```

Hovedstaden, reverse geocoded using `geom:` coordinates:

```
curl -s 'http://localhost:1111/country?latitude=55.690196&longitude=12.89523' | python -mjson.tool
[]
```

Hovenstaden, reverse geocoded _for region_ using `geom:` coordinates:

```
curl -s 'http://localhost:1111/region?latitude=55.690196&longitude=12.89523' | python -mjson.tool
[
    {
        "Id": 85688377,
        "Name": "Sk\u00e5ne",
        "Offset": -1,
        "Placetype": "region"
    }
]
```

Hovedstaden, reverse geocoded _for region_ using `lbl:` coordinates:

```
curl -s 'http://localhost:1111/region?latitude=55.697507&longitude=12.490749' | python -mjson.tool
[
    {
        "Id": 85682581,
        "Name": "Hovedstaden",
        "Offset": -1,
        "Placetype": "region"
    }
]
```

Hovedstaden and Skåne are adjacent to one another so this is basically a good candidate for sanity checking things. The functional equivalent of this during the Flickr years were El Paso (TX) and Cuidad Juarez (MX).

Basically what happens is we try (endeavour) to have confidence in the higher level hierarchies because there are fewer of them and national borders are relatively fixed and efficiencies (fewer reverse geocoding requests). Mostly it works but sometimes this.

Honestly, I thought everything had been Mapshaper-ed by the time we rebuilt the hierarchies for Stephen but it's possible that we weren't yet giving precedence to 'lbl:` centroids.

So, yes you are correct that descendants need to be updated. The bad news is that's a bit annoying. The worse news is this won't be the only time we need to do this. The good news is that it's a relatively small dataset and it will be a good exercise to test/polish the tools for doing this:

https://whosonfirst.mapzen.com/spelunker/id/85682581/descendants/

I am going to finish waking up before I try doing this, though. It should all be squared away by lunch or mid-afternoon, barring any unforeseen bad crazines...

**Comment by orangejulius** [comment]
Yeah, it's definitely a centroid thing. Do wake up before actually changing anything. We can hotfix this in Pelias so don't worry: you aren't blocking us from fixing things for search :)

**Comment by nvkelso** [comment]
Please let's fix the data and not add bandaid code that will become outdated.

**Comment by thisisaaronland** [comment]
Oh don't worry, we'll fix the data. The fix in this one instance is relatively straightforward.

Operationalizing it (for the next time) is experiencing some code-turbulence around placetypes and their related hierarchies. We've never had a tangible reason to address these issue until now and I am trying to see whether this is a reasonable forcing-function.

If it starts smelling like yaks, I will just fix things the manual/dumb/simple way.

**Comment by thisisaaronland** [comment]
screen shot 2016-04-22 at 13 38 34

Closing because this got sorted as part of issue #255 (which is still open pending venues getting sorted).
